docs/README.md

```diff
@@ -21,6 +21,8 @@ installed. Also install the following libraries:
     # Following is needed only for generating API docs
     $ sudo pip install sphinx
     $ sudo Rscript -e 'install.packages(c("knitr", "devtools", "roxygen2", "testthat"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo Rscript -e 'install.packages(c("rmarkdown"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo pip install pandoc pandoc-citeproc
```

Comment:
I agree with that, but, now we still have two additional commands here, when they can just be packages installed by the commands above. That was my earlier point.